### ISSUE TITLE
WindowsError while running psort with analysis plugins

### ISSUE THREAD

**Issue by dc3-plaso** [log2timeline/plaso#972]
**Plaso version:**

Current HEAD: https://github.com/log2timeline/plaso/commit/d007b96bb6332b3e355551ce3afe15bc3af0c6e6

**Operating system Plaso is running on:**

Windows 7 64bit

**Installation method:**

Development environment.

**Description of problem:**

Psort produces a WindowsError when trying to use an analysis plugin. (It does not happen when just using the output modules.)

Note: This same error also happened to log2timeline before I updated the master branch. (It was about a month behind.)

**Debug output/tracebacks:**

```
(plaso) C:\dev\plaso>python tools\psort.py -o xlsx G:\test_data\set_1_output.zip -w G:\test_data\set_1_output.xlsx --status_view none -d --analysis browser_search
2016-08-30 12:37:21,882 [INFO] (MainProcess) PID:9376 <psort> Starting analysis plugins.
2016-08-30 12:37:21,888 [DEBUG] (MainProcess) PID:9376 <zeromq_queue> Creating socket for browser_search output event queue
2016-08-30 12:37:21,898 [DEBUG] (MainProcess) PID:9376 <zeromq_queue> browser_search output event queue bound to random port 61649
2016-08-30 12:37:22,417 [INFO] (MainProcess) PID:9376 <psort> Started analysis plugin: browser_search (PID: 14200).
2016-08-30 12:37:24,822 [INFO] (MainProcess) PID:9376 <psort> Analysis plugins running
Traceback (most recent call last):
  File "tools\psort.py", line 867, in <module>
    if not Main():
  File "tools\psort.py", line 853, in Main
    tool.ProcessStorage()
  File "tools\psort.py", line 787, in ProcessStorage
    status_update_callback=status_update_callback)
  File "C:\dev\plaso\plaso\frontend\psort.py", line 103, in AnalyzeEvents
    status_update_callback=status_update_callback)
  File "C:\dev\plaso\plaso\multi_processing\psort.py", line 536, in AnalyzeEvents
    storage_writer.Open()
  File "C:\dev\plaso\plaso\storage\zip_file.py", line 3217, in Open
    self._storage_file.Open(path=self._output_file, read_only=False)
  File "C:\dev\plaso\plaso\storage\zip_file.py", line 2754, in Open
    self._OpenZIPFile(path, read_only)
  File "C:\dev\plaso\plaso\storage\zip_file.py", line 1740, in _OpenZIPFile
    os.rename(path, zipfile_path)
WindowsError: [Error 32] The process cannot access the file because it is being used by another process

```

**Source data:**

Small plaso storage file of chat message events.

**Comment by joachimmetz** [comment]
Maybe related: https://codereview.appspot.com/306360043/

**Comment by joachimmetz** [comment]
Assigning to @Onager since there might be pending change to fix this issue.

**Comment by Onager** [comment]
Probably not related to that change, but I suspect this might be environmental, possibly some other program (antivirus or a backup agent) accessing the file when we're trying to rename it.

**Comment by dc3-plaso** [comment]
@Onager I've tried your patch, and it still doesn't work. 

Things I've tried:
- Rebooted the system and then just ran psort. (There are no antivirus programs or anything that would have a handle on the file.)
- I've tried changing the `os.rename(path, zipfile_path)` to something like `os.rename(path, path + '_test')` in case it was something about Windows not liking that is was in a directory.
- Changed them to absolute paths
- Used the null output module.

Let me know if there is anything else I should try.
